frontend/src/components/immunohistochemistry/ImmunohistochemistryDashboard.js

```diff
@@ -284,6 +285,7 @@ function ImmunohistochemistryDashboard() {
     <>
       {notificationVisible === true ? <AlertDialog /> : ""}
       {loading && <Loading description="Loading Dasboard..." />}
+      <GenericHomeBreadCrumb/>
```

Comment:
My thinkin was that this componenet should be able to take in a prop
like this 
```
const breadcrumbData = [
  { path: '/', title: intl.formatMessage({ id: "home.label" }) },
  { path: '/ResultValidation', title: intl.formatMessage({ id: "sidenav.label.validation" }) }
```


How are you going to pass in the different paths ??

Reply:
Was thinking of hard coding all of them in the breadcrumbData array for starters.